app.go

```diff
@@ -26,14 +26,23 @@ func app(cmd *cobra.Command, args []string) {
 	var err error
 
 	// Get version of Docker benchmark to run
-	if dockerVersion != "" {
-		version = dockerVersion
+	if benchmarkVersion != "" {
+		version = benchmarkVersion
```

Comment:
I think you also need to test whether `dockerVersion` is empty string - if the user specifies both dockerVersion and benchmarkVersion it needs to return an error

Reply:
Do we need to return error? I think that currently if the user specifies both then the benchmark will be the one to take effect, maybe we should print the error for logging? (but still run it)

Reply:
We should be consistent with kube-bench, and that errors if you try to specify both. See the help text at https://github.com/aquasecurity/kube-bench/blob/58bea9c89b9be43ebdf76de256d916a608df43c4/cmd/root.go#L176 (and also the help text you've used in root.go below!)